source/common/protobuf/utility.h

```diff
@@ -48,10 +49,26 @@ class RepeatedPtrUtil {
 class MessageUtil {
 public:
   static std::size_t hash(const Protobuf::Message& message) {
-    return std::hash<std::string>{}(message.SerializeAsString());
+    std::string text;
```

Comment:
Unrelated, but it was causing string flakes, since the non-deterministic proto serialization led to distinct hashes for the same proto.